packages/video_player/video_player/lib/video_player.dart

```diff
@@ -247,13 +247,14 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   ///
   /// This will load the file from a file:// URI constructed from [file]'s path.
```

Comment:
This comment needs to be updated to explain what `httpHeaders` does in this context, since it's non-obvious.